languages/csharp/src/main/java/de/jplag/csharp/CSharpLanguage.java

```diff
@@ -25,7 +25,7 @@ public String[] suffixes() {
 
     @Override
     public String getName() {
-        return NAME;
```

Comment:
I would suggest either removing the name constant or changing the constant itself

Reply:
Ok I made one commit right now for this.